pure_pursuit_wrapper/src/pure_pursuit_wrapper.cpp

```diff
@@ -86,34 +229,37 @@ void PurePursuitWrapper::trajectoryPlanHandler(const cav_msgs::TrajectoryPlan::C
     }
   }
 
-  std::vector<double> lag_speeds = apply_response_lag(speeds, downtracks, config_.vehicle_response_lag); // This call requires that the first speed point be current speed to work as expected
-
-  autoware_msgs::Lane lane;
-  lane.header = tp->header;
-  std::vector<autoware_msgs::Waypoint> waypoints;
-  waypoints.reserve(trajectory_points.size());
-
-  for (int i = 0; i < trajectory_points.size(); i++)
+  std::vector<double> lag_speeds;
+  if (config_.is_delay_compensation)
```

Comment:
Isn't this parameter an autoware auto param? Do they do the same thing? Should we differentiate between the two?

Reply:
They are different things. We should apply the vehicle_response_lag nonetheless, so I removed this requirement. This delay accounts for the delay between when the trajectory is generated and when the cmd is generated using them.